webpack.config.js

```diff
@@ -1,27 +1,59 @@
 const webpack = require('webpack');
-const {join} = require('path');
+
+class FallbackRequireWebpackPlugin {
+  apply(compiler) {
+	compiler.hooks.compilation.tap('FallbackRequireWebpackPlugin', (compilation) => {
+		const {renderRequire} = webpack.javascript.JavascriptModulesPlugin.getCompilationHooks(compilation);
+		renderRequire.tap(
+			'FallbackRequireWebpackPlugin',
+			(content, renderContext) => content.replace(
+				/\s*return module\.exports;\s*/,
+				'\nreturn (module.exports && module.exports.name === "webpackEmptyContext") ? require(moduleId) : module.exports;',
+			),
+		);
+	});
+  }
+}
 
 module.exports = {
+	mode: 'production',
 	target: 'node',
 	context: __dirname,
-	entry: './src/plop.js',
+	entry: {
+		main: {
+			import: './src/plop.js',
+			filename: './dist/plop.js',
+		},
+		bin: {
+			dependOn: 'main',
+			import: './src/bin/plop.js',
+			filename: './bin/plop.js',
+		},
+	},
 	output: {
-		path: join(__dirname, 'dist'),
-		filename: 'plop.js',
-		libraryTarget: 'commonjs2',
```

Comment:
`libraryTarget: 'commonjs2'` does not work with multiple entrypoints. It's probably an upstream webpack bug, but I've read soooo much Webpack source code this weekend so no issue to point to.

The error I got was this, meaning the exports of the commonjs2 module were overriding the exports of the runtime webpack code.

```
TypeError: __webpack_require__.C is not a function
```